src/components/Covid.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+
+
+const Covid = () => {
+
+    const [data, setData] = useState(null);
+
+    useEffect(() => {
+        axios.get('https://api.covid19api.com/summary')
+            .then(response => setData(response.data))
+            .catch(error => console.log(error))
+    }, [])
+
+
+    if (data) {
+        const dataSort = data.Countries.sort((a, b) => {
+            if (a.TotalConfirmed > b.TotalConfirmed) {
+                return -1;
+            }
+            if (a.TotalConfirmed < b.TotalConfirmed) {
+                return 1;
+            }
+            return 0;
+
+        })
+
```

Comment:
It is not clear what is this logic there, if is doing something to the data so it should be inside of the useEffect function